fieldsolver/gridGlue.hpp

```diff
@@ -186,5 +186,62 @@ template< unsigned int numFields > void getFieldDataFromFsGrid(
          }
       }
    }
+}
 
+/*Compute coupling DCCRG <=> FSGRID 
+
+  onDccrgMapRemoteProcess   maps fsgrid processes (key) => set of dccrg cellIDs owned by current rank that map to  the fsgrid cells owned by fsgrid process (val)
+
+  onFsgridMapRemoteProcess  maps dccrg processes  (key) => set of dccrg cellIDs owned by dccrg-process that map to current rank fsgrid cells 
+  onFsgridMapCells          maps remote dccrg CellIDs to local fsgrid cells
+*/
+
+template <typename T, int stencil> void computeCoupling(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
+      const std::vector<CellID>& cells,
+      FsGrid< T, stencil>& momentsGrid,
```

Comment:
Here this should be renamed to technicalgrid as that's the one it now gets.
Do we even want to pass an explicit coupling as a parameter, if it's included in the technicalgrid anyway?